configurations/30-sl-nginx.entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+ME=$(basename "$0")
+DEFAULT_CONF_FILE="/etc/nginx/conf.d/simplelogin.conf"
+if [ -f $DEFAULT_CONF_FILE ]; then
+    echo "$ME: info: Simplelogin Nginx configuration file is already exists."
+    exit 0
+fi
+echo "$ME: info: No configuration found. Going to create one."
+cat <<EOT > $DEFAULT_CONF_FILE
+server {
+    listen       80;
+    listen  [::]:80;
+    server_name  sl.doanguyen.com;
```

Comment:
I wonder if the `server_name` can be extracted from an environment variable here?

Reply:
sure, i'm going to update it